dashboard/package.json

```diff
@@ -84,6 +84,7 @@
     "protractor-jasmine2-screenshot-reporter": "^0.3.3",
     "proxy-middleware": "^0.15.0",
     "raw-loader": "^0.5.1",
+    "regenerator-runtime": "^0.13.3",
```

Comment:
do we have CQs for these new dependencies ?

Reply:
Not yet, I'm not a Che commiter so someone else has to submit the CQ's

Reply:
@JPinkney did we ever file that CQ?

Reply:
I don't think we filed it. I'm not sure what determines which packages we need to file cq's for but this is a list of the new dependencies with their licenses: 

regenerator-runtime - MIT
glob - ISC
monaco-editor-core - MIT
monaco-languageclient - MIT
monaco-languages - MIT
umd-compat-loader - Apache-2.0
vscode-json-languageservice - MIT
vscode-languageserver-types - MIT
yaml-language-server - MIT

Reply:
created